docs/user_manual/type_data.md

```diff
@@ -0,0 +1,514 @@
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Library Types" %}
+
+Library type is a classifier for libraries that depends on the platform type. Library types for Illumina libraries
+include 'Paired End,' 'Single End,' and 'Mate Pair,' for example. Library types may be archived so that the option is
+no longer available for new libraries. Modifications to library types can only be made via direct access to the MISO
+database. As such, they must be performed by a MISO adminstrator.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Library Designs" %}
+
+Note: This item only applies if your site uses [detailed sample mode](site_configuration.html#detailed_sample_mode).
+
+A library design is a grouping of [selection](#library_selection_types) and [strategy](#library_strategy_types) types.
+The library designs that are available for a library depend on the sample class of the aliquot that the library was
+propagated from. Modifications to library designs can only be made via direct access to the MISO database. As such,
+they must be performed by a MISO adminstrator.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Library Selection Types" %}
+
+Library selection type is defined by [NCBI](https://www.ncbi.nlm.nih.gov/) as the "method used to enrich the target in
+the sequence library preparation." This property is required in order to submit libraries to the SRA (see the
+[SRA section](sra.html)). The values available in MISO should be limited to the default options, which are the same as
+defined in the SRA schema.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Library Strategy Types" %}
+
+Library strategy type is defined by [NCBI](https://www.ncbi.nlm.nih.gov/) as the "sequencing type intended for this
+library." This property is required in order to submit libraries to the SRA (see the [SRA section](sra.html)). The
+values available in MISO should be limited to the default options, which are the same as defined in the SRA schema.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Kit Descriptors" %}
+
+A kit descriptor describes a kit product that is used for library or run preparation. The types of kits available in
+MISO are:
+
+* Library kit - used for libraries
+* Sequencing kit - not currently used in MISO
+* Clustering kit - used for sequencing containers
+* Multiplexing kit - used for sequencing containers
+* Extraction kit - not currently used in MISO
+
+Modifications to kit descriptors can only be made via direct access to the MISO database. As such, they must be
+performed by a MISO adminstrator.
```

Comment:
Users can add and edit kit descriptors (called "Kits" in the List page, and "Kit Descriptor" in the Edit page...) from the List Kits page.

Reply:
It's true, we would! but the functionality exists in the GUI. I'm ok with leaving it like this though if the document's primary goal is to support our very-real users, with a secondary goal of supporting hypothetical external institute users.

Reply:
The goal is to document everything that MISO does and everything that can be done in MISO, so I'm updating to give details of how to add kit descriptors in MISO

Reply:
Yeah, that's our convention. They could do it if they wanted, but they can't add targeted sequencing via the UI.